src/transactions/reducer.ts

```diff
@@ -22,7 +22,7 @@ export interface InviteTransactions {
 type TransactionsByNetworkId = {
   [networkId in NetworkId]?: TokenTransaction[]
 }
-interface State {
+export interface State {
```

Comment:
nit: It seems this is exported only for testing.
We could infer it instead for the test?